modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -36,7 +36,8 @@ class AutoEstablishedClaim < ApplicationRecord
 
     EVSS_CLAIM_ATTRIBUTES = %i[date_filed min_est_date max_est_date open waiver_submitted
                                documents_needed development_letter_sent decision_letter_sent
-                               requested_decision va_representative].freeze
+                               requested_decision va_representative current_phase_back ever_phase_back
+                               latest_phase_type phase_change_date].freeze
```

Comment:
Can you help me understand why this change is needed?

Reply:
Yes, so this was one of the many changes from Kayla. She wants to provide the actual status or phase from BGS, rather than the condensed phases that we normally use. So in the show, more detail.
The initialize is called from two places now, one just needs the normal phase, the other needs to derive the phase from the `phase_type_change_ind`, which is an int, and the mapper converts that int to a BGS phase.
Does my explanation make sense? Do you have a preferred way to handle this scenario?

Reply:
Ok.  I guess more specifically, I'm wondering if this change actually changes anything?
From what I can tell, it looks like the only place `EVSS_CLAIM_ATTRIBUTES` is ever referenced in `vets-api` is directly below this ([right here](https://github.com/department-of-veterans-affairs/vets-api/blob/93e17dae18113b0d3df2447224e25a4d5c7814c4/modules/claims_api/app/models/claims_api/auto_established_claim.rb#L44)).  And it looks like all it does is create a method for each attribute that returns `nil`.  (I honestly have no idea why that even exists 😂) If you remove this change, does it affect any behavior or anything in your PR?

Reply:
omg, when I clicked on `view changes`, it took me somewhere else. No, this is not needed, and I will remove it now!